docs/docs/external/getting-started.md

```diff
@@ -78,8 +80,9 @@ Using the sequence of instructions below, you can start two separate Fabric netw
 
 A library, as companion to the `hyperledger/fabric-sdk-node`, is defined in the `sdks/fabric/interoperation-node-sdk` folder. This contains functions for Fabric Gateway-based applications to exercise interoperation capabilities via relays and also a number of utility/helper functions. The Fabric-CLI tool, which we will use later, depends on this library.
 
-To build the library, do the following:
+(OPTIONAL) To build the library, do the following:
```

Comment:
I think we should expand this document a bit more. The given instructions are for running relays and drivers in the host, but we can add sections to install and run them in containers. Up front, we can say, if you wish to run in the host, go to section X and if you wish to run in a container, go to section Y.